src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -42,6 +42,10 @@ public static IEnumerable<object[]> SerializableEqualityComparers_MemberData()
         /// </summary>
         public static IEnumerable<object[]> SerializableObjects_MemberData()
         {
+            // Save old culture and set a fixed culture for object instantiation
+            CultureInfo oldCulture = CultureInfo.CurrentCulture;
+            CultureInfo.CurrentCulture = new CultureInfo("en-US");
```

Comment:
As this is an iterator, control returns to the caller at each yield point, and we don't know what xunit does or doesn't do between each yield.  That means we're changing the culture on a thread that will then potentially be used for something else.  If you want to do it this way, we should change SerializableObject_MemberData to not be an iterator, e.g. just have it return and return an object[][] populated with all of these entries, rather than yielding them.  Then you can set the culture before, and unset it after in a finally block, and no one else will use the thread in the interim.

That said, I was under the impression this setting of culture like this was a no-no on uapaot, as it currently has process-wide impact, which is why we use RemoteInvoke for these.  Has that been fixed, or if not, why is this ok in this case?
cc: @tarekgh